src/components/ThemeProvider/__stories__/ThemeProvider.stories.tsx

```diff
@@ -0,0 +1,179 @@
+import cx from "classnames";
+import React from "react";
+import ThemeProvider, { ThemeProviderProps } from "../ThemeProvider";
+import Button from "../../Button/Button";
+import { SystemTheme, ThemeColor } from "../ThemeProviderConstants";
+import Dropdown from "../../Dropdown/Dropdown";
+import Flex from "../../Flex/Flex";
+import {
+  crmProductTheme,
+  marketingProductTheme,
+  projectManagementProductTheme,
+  softwareProductTheme
+} from "./product-themes";
+import styles from "./ThemeProvider.stories.module.scss";
+
+export const ThemeProviderTemplateOverview = (args: JSX.IntrinsicAttributes & ThemeProviderProps) => {
+  return (
+    <>
+      <ThemeProvider {...args}>
+        <Button>Themed</Button>
+      </ThemeProvider>
+    </>
+  );
+};
+
+export const ThemeProviderThemingScopeTemplate = (_args: JSX.IntrinsicAttributes & ThemeProviderProps) => {
+  return (
+    <>
+      <ThemeProvider
```

Comment:
This happens when I clear the theme via the storybook toggle. which means we might need to consider a fallback regarding how to handle the case a specific theme is not applied
<img width="1092" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/191357af-538a-41b3-b134-3520c8799711">

Reply:
What kind of fallback you are thinking about? When systemTheme is not defined, ThemeProvider sysyemTheme dependent color overrides are not applying - that's the design
In the Monolith PR, though, I've implemented a separate logic, which works for default-system-theme as well